common/lib/xmodule/xmodule/tests/test_lti_unit.py

```diff
@@ -173,7 +172,7 @@ def test_real_user_is_none(self):
         """
         self.xmodule.verify_oauth_body_sign = Mock()
         self.xmodule.has_score = True
-        self.system.get_real_user = Mock(return_value=None)
+        self.system.service(self.xmodule, 'user').user = None
```

Comment:
Hmm, does the `UserService` have a `user` property?

Reply:
No -- this was just my shortcut to getting a `None` user returned by the `StubUserService.get_user_by_anonymous_id`. I'll find a better way.